docs/reference/supported-environments.md

```diff
@@ -57,8 +58,9 @@ This overview shows which Zeebe version works with which Modeler, Operate, Taskl
 | Desktop Modeler 4.11+ | Zeebe 1.2.x | Operate 1.2.x Tasklist 1.2.x IAM 1.2.x      | -              |
 | Desktop Modeler 4.12+ | Zeebe 1.3.x | Operate 1.3.x Tasklist 1.3.x IAM 1.3.x      | Optimize 3.7.x |
 | Desktop Modeler 5.0+  | Zeebe 8.0.x | Operate 8.0.x Tasklist 8.0.x Identity 8.0.x | Optimize 3.8.x |
+| Web Modeler (Beta)    | Zeebe 8.1.x | Operate 8.1.x Tasklist 8.1.x Identity 8.1.x | Optimize 3.9.x |
 
-_Note: You can use newer Modeler versions with older Zeebe versions too._
+_Note: You can use newer versions of the Desktop Modeler and the Web Modeler with older Zeebe versions too._
```

Comment:
@CatalinaMoisuc Just to clarify here -- do we have a preference on "**the** Desktop/Web Modeler" or just "Web/Desktop Modeler?" In existing documentation, we've just utilized "Use newer versions of Desktop Modeler and Web Modeler..." for example.

Reply:
My personal preference is for Dektop/Web Modeler without the article. 

Happy to learn about the definitive answer to your question.

Reply:
Agreed here @pihme -- I'd be happy to launch a PR to ensure this form of reference is how Modeler is used across docs 👍

Reply:
We refer to Web and Desktop Modeler without an article (note that we also sometimes refer to Camunda Modeler when we don't separate between Web and Desktop Modeler, or sometimes we even refer to Desktop Modeler with it—since this is it's "legacy" name

Reply:
Excellent. Thanks for the confirmation @christian-konrad -- I'll ensure this is also clarified in our style guide. Mentioning [the PR](https://github.com/camunda/camunda-platform-docs/pull/1398) where I've adjusted these references across our docs as well.